apps/asap-server/scripts/create-algolia-backup.js

```diff
@@ -0,0 +1,28 @@
+const algoliasearch = require('algoliasearch');
+const fs = require('fs/promises');
+
+const main = async () => {
+  console.log({
+    appId: process.env.ALGOLIA_APP_ID,
+    apiKey: process.env.ALGOLIA_API_KEY,
+  });
+
+  const index = algoliasearch(
+    process.env.ALGOLIA_APP_ID,
+    process.env.ALGOLIA_API_KEY,
+  ).initIndex(process.env.ALGOLIA_INDEX);
+
+  let hits = [];
+
+  await index.browseObjects({
```

Comment:
Won't this consume a lot of memory? What happened to the copy index function?

Reply:
well copying index is copying index and this script serves a different purpose which is to create a db dump into a file

Reply:
but to answer your other question - it probably will, i could batch it and use streams but for that we'd need a writeable object/array type of stream and it just seemed a little unnecessary given that we dont really have that many research outputs. Do you think i should look into it anyway?

Reply:
I think it would be better for it not to fail at somepoint in the future